app/src/main/contrib/components/Responsibilities/ModelRunParameters/ModelRunParametersSection.tsx

```diff
@@ -1,73 +1,21 @@
-import {ModelRunParameterSet} from "../../../../shared/models/Generated";
 import * as React from "react";
-import {Alert} from 'reactstrap';
-import {longestTimestamp} from "../../../../shared/Helpers";
-import {Form} from "../../../../shared/components/Form";
-import {CustomFileInput} from "../../../../shared/components/CustomFileInput";
-import {ModelRunParameterDownloadCertificate} from "./ModelRunParameterDownloadCertificate";
+import {ModelRunParametersStatus} from "./ModelRunParametersStatus";
+import {ModelRunParametersForm} from "./ModelRunParametersForm";
 
 interface Props {
     url: string;
     disease: string;
-    sets: ModelRunParameterSet[];
 }
 
-interface State {
-    fileInputKey: Date;
-}
-
-export class ModelRunParametersSection extends React.Component<Props, State> {
-
-    constructor() {
-        super();
-        this.state = {
-            fileInputKey: new Date()
-        }
-    }
-
-    onSuccess() {
-        this.setState({
-            // this is to trick React into re-rendering the file input when the form has been successfully submitted
-            // see https://github.com/erikras/redux-form/issues/769
-            fileInputKey: new Date()
-        })
-    }
+export class ModelRunParametersSection extends React.Component<Props, undefined> {
 
     render(): JSX.Element {
 
-        let alertContent = <span>You have not uploaded any parameter sets for {this.props.disease}</span>;
```

Comment:
this component has been split into 2 - `ModelRunParametersStatus` and `ModelRunParametersForm`